packages/snaps-execution-environments/src/common/commands.ts

```diff
@@ -44,7 +47,23 @@ export function getHandlerArguments(
         transactionOrigin,
       };
     }
+    case HandlerType.OnNameLookup: {
+      assertIsOnNameLookupRequestArguments(request.params);
+
+      // TS complains that domain/address are not part of the type
+      // casting here as we've already validated the request args in the above step.
+      const { chainId, domain, address } = request.params as any;
```

Comment:
Can we cast to a type other than `any`?

Reply:
I don't think so

Reply:
I mean, cast to an actual type that has these properties 😅

Reply:
lol ohh, coming right up.